CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # CHANGELOG
 
+## 1.2.5 (2020-05-25)
```

Comment:
2020-06 :)

Btw, better not to change any CHANGELOG on a PR, since multiple PRs can happen simultaneously.